src/modules/bill-runs-import/lib/import.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const queries = require('./queries');
+const slack = require('../../../lib/slack');
+const { pool } = require('../../../lib/connectors/db');
+const { logger } = require('../../../logger');
+
+const createRow = (tableName, query) => ({
+  tableName,
+  query
+});
+
+const importQueries = [
+  createRow('NALD_BILL_RUNS', queries.importNaldBillRuns)
+];
+
+/**
+ * Logs a message and posts to Slack
+ * @param {String} str - the message
+ */
+const log = str => {
+  const message = `Bill run import: ${str}`;
+  logger.info(message);
+  slack.post(message);
+};
+
+/**
+ * Run SQL queries to import bill runs to the water
+ * billing tables.
+ * It is envisaged this will only be run once in production
+ *
+ * @return {Promise}
+ */
+const importBillRuns = async () => {
+  try {
+    log('Starting...');
```

Comment:
I would suggest making this message:

1. A little more verbose, such as `Billing Batches import from NALD: Started` or something to that effect

2. Posting the message to Slack (So that we can see this happening in production when the time comes)

Reply:
It should already:
* prefix all messages with "Bill run import:"
* send messages to Slack